core/Site.php

```diff
@@ -99,24 +123,6 @@ protected static function setSite($idSite, $infoSite)
             throw new UnexpectedWebsiteFoundException("An unexpected website was found, check idSite in the request.");
         }
 
-        /**
-         * Triggered so plugins can modify website entities without modifying the database.
-         *
-         * This event should **not** be used to add data that is expensive to compute. If you
-         * need to make HTTP requests or query the database for more information, this is not
-         * the place to do it.
-         *
-         * **Example**
-         *
-         *     Piwik::addAction('Site.setSite', function ($idSite, &$info) {
-         *         $info['name'] .= " (original)";
-         *     });
-         *
-         * @param int $idSite The ID of the website entity that will be modified.
-         * @param array $infoSite The website entity. [Learn more.](/guides/persistence-and-the-mysql-backend#websites-aka-sites)
-         */
-        Piwik::postEvent('Site.setSite', array($idSite, &$infoSite));
```

Comment:
Here we will break API, it was only there for `MetaSites` plugin but in theory other plugins could have used it as well. We spend I think > 7 seconds just here in triggering this event. We decided to deprecate it. I will need to adjust MetaSites plugin, already started doing it.